File: 4th Semester/Factorial.java
/* Factiorial Number */

public class Factorial {

    public static void main(String[] args) {
        int n = 6;
        int fact=1;

        for(int i=1;i<=n;i++){
            //Factorial formula
            fact = fact*i;
        }

        System.out.println("Factorial of 6 is: "+fact);
    }

}

File: Loop/03. Factorial/factorial.c
//factorial of a given number
#include<stdio.h>
int main()
{
    int n,i,fact=1;
    printf("Enter the number\n");
    scanf("%d",&n);
    for(i=1;i<=n;i++)
    {
        fact=fact*i;
    }
    printf("Factorial of %d is %d\n",n,fact);
    return 0;
}